lib/datadog/kit/events.rb

```diff
@@ -0,0 +1,31 @@
+# typed: false
+
+module Datadog
+  module Kit
+    # Tracking events
+    module Events
+      def self.track_login(trace, id:, success:, **others)
+        subtag = success ? 'success' : 'failure'
+        event = "users.login.#{subtag}"
+
+        track(event, trace, **others)
+
+        if success
+          trace.set_tag('usr.id', id)
+        else
+          trace.set_tag("appsec.events.users.login.#{subtag}.usr.id", id)
+        end
+      end
+
+      def self.track(event, trace, **others)
+        trace.set_tag("appsec.events.#{event}.track", true)
```

Comment:
```suggestion
        trace.set_tag("appsec.events.#{event}.track", 'true')
```

Because `#set_tag` will always call `to_s`, this making it a string will save a bit on processing.